src/runtime/index.js

```diff
@@ -14,7 +14,11 @@ const { parseRunParams } = require('./params')
  * @returns {RunEnv}
  */
 function currentRunEnv () {
-  return parseRunEnv(process.env)
+  if (process.title === 'browser') {
+    return parseRunEnv(window.testground.env)
+  } else {
+    return parseRunEnv(process.env)
+  }
```

Comment:
Is this standard? Else, could we simplify with something like `process.env || window.testground.env`?

Reply:
Sure thing.

Reply:
Will drop this in favour of the browser spec.